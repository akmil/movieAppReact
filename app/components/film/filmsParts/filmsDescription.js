import React from 'react';

// Description
export const FilmsDescription = ({ description }) => (
    <div className="film-description">{description}</div>
);
