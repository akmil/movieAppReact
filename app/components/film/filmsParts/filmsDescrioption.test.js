import React from 'react';
import {FilmsDescription} from './filmsDescription';
import renderer from 'react-test-renderer';

describe('<films FilmsDescription />', () => {
    it('should render SearchButton without errors', () => {
        const wrapper = renderer.create(<FilmsDescription/>);
        expect(wrapper).toMatchSnapshot();
    })
});