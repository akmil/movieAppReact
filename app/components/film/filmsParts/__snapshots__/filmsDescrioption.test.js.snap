// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<films FilmsDescription /> should render SearchButton without errors 1`] = `
<div
  className="film-description"
/>
`;
